server/jetstream.go

```diff
@@ -474,7 +473,7 @@ func (a *Account) EnableJetStream(limits *JetStreamAccountLimits) error {
 		}
 
 		stats := mset.Stats()
-		s.Noticef("  Restored %s messages for MsgSet %q", humanize.Comma(int64(stats.Msgs)), fi.Name())
+		s.Noticef("  Restored %d messages for MsgSet %q", stats.Msgs, fi.Name())
```

Comment:
I liked the comma version here when messages get big. Can we snapshot that from humanize (giving credit of course)?

Reply:
I guess :) this even used to be a debug line, but sure let me see about writing some helper

Reply:
When I was doing my large tests, with 1B+ it was vert helpful.

Reply:
thats fair, I'll write a helper

Reply:
Not really sure what the right way is to give credit when pulling in a function like this let me know if something else is better - but there it is borrowed from humanize.